sktime/transformations/series/summarize.py

```diff
@@ -453,55 +459,54 @@ class description for in-depth explanation.
         if isinstance(Z, pd.core.groupby.generic.SeriesGroupBy):
```

Comment:
I think the issue with the failing tests is located in this if condition. 

A rolling - summarize operation and a SeriesGroupBy object seems to double the index level 0. Not sure why. Thus, the easiest fix would be to drop the first index in 474. However, this is only fixing the symptom and not really the cause... Thus @fkiraly do you have a better idea.